controllers/datafoundation.go

```diff
@@ -71,21 +73,30 @@ type dataFoundationReconciler struct {
 //+kubebuilder:rbac:groups=network.openshift.io,resources=ingressnetworkpolicies,verbs=create;get;list;watch;update
 //+kubebuilder:rbac:groups=operators.coreos.com,resources=clusterserviceversions,verbs=get;list;watch;delete;update;patch
 
-func dfSetupWatches(controllerBuilder *builder.Builder) {
+func dfSetupWatches(controllerBuilder *builder.Builder, offeringReconciler *ManagedFusionOfferingReconciler) {
```

Comment:
It is common to call the governing reconciler `r` and have it as the first parameter (to follow all the other 
```suggestion
func dfcSetupWatches(r *ManagedFusionOfferingReconciler, controllerBuilder *builder.Builder) {
```